domain/schema/model.go

```diff
@@ -152,11 +152,6 @@ func ModelDDL() *schema.Schema {
 			"OLD.subject_type_id <> NEW.subject_type_id OR OLD.scope_uuid <> NEW.scope_uuid OR OLD.scope_type_id <> NEW.scope_type_id",
 			"secret permission subjects and scopes must be identical",
 		),
-
-		triggerGuardForTable("sequence_charm_local",
-			"OLD.reference_name = NEW.reference_name AND NEW.sequence <= OLD.sequence",
-			"sequence number must monotonically increase",
-		),
```

Comment:
Can we instate this for the new sequence table?

Reply:
Yeah good idea

Reply:
Done